validator_json/sigma.json

```diff
@@ -1876,6 +1876,29 @@
         "custom": []
       }
     },
+    "windows_none_iis-configuration": {
+      "logsource": {
+        "product": "windows",
+        "category": null,
+        "service": "iis-configuration",
+        "definition": null
+      },
+      "description": null,
+      "field": {
+        "natif": [
```

Comment:
I would call this "native" if this is meant.

Reply:
done